bazel/src/test/data/googleapis/google/example/library/v1/BUILD.bazel.baseline

```diff
@@ -97,6 +97,12 @@ java_gapic_assembly_gradle_pkg(
     ],
 )
 
+alias(
```

Comment:
Too verbose, adds aliasing to solve the non-pretty name issue. This is too verbose.